src/Mod/Path/Gui/Resources/preferences/PathJob.ui

```diff
@@ -655,6 +655,9 @@
        </item>
        <item>
         <widget class="QCheckBox" name="toolsOpenLastLibrary">
+         <property name="toolTip">
+          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Causes the toollibrary manager to remember last directory.   This ONLY affects legacy tools. &lt;/p&gt;&lt;p&gt;This control will be deprecated and removed in future version&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+         </property>
          <property name="text">
           <string>Remember last library</string>
```

Comment:
Maybe even add "(legacy tools only)" or "(deprecated)" to this string?

Reply:
Yep. thanks